garak/probes/latentinjection.py

```diff
@@ -602,20 +579,17 @@ class LatentWhois(Probe, LatentInjectionMixin):
     payload_triggers = ["not malicious"]
 
     def __init__(self, config_root=_config):
-        super().__init__(config_root)
+        Probe.__init__(self, config_root)
         self._build_prompts_triggers(cap=self.soft_probe_prompt_cap)
 
-    def _attempt_prestore_hook(self, attempt: Attempt, seq: int) -> Attempt:
-        return LatentInjectionMixin._attempt_prestore_hook(self, attempt, seq)
 
-
-class LatentWhoisSnippet(LatentInjectionFactSnippetEiffelFull):
+class LatentWhoisSnippetFull(FactSnippetMixin, LatentWhois):
```

Comment:
Optional, this does not look be needed, nothing is being inherited from `FactSnippetMixin`.

`FactSnippetMixin.injection_instructions` is still accessible as written in line 602 if this is removed.

```suggestion
class LatentWhoisSnippetFull(LatentWhois):
```

Reply:
Snippet building needs refactoring anyway after the current fix in #1181 - will update that PR and then it should be processed directly after this one.

Reply:
on second thoughts:
- will leave this inheritance to express intent
- #1181 creeps if it addresses this
- will pick up in later refactor-only PR that needn't block recalibration